### ISSUE TITLE
Wrench-trains

### ISSUE THREAD

**Issue by andrejsharapov** [Templarian/MaterialDesign#7213]
### Before submitting a new icon contribution, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://pictogrammers.com/library/mdi) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](https://pictogrammers.com/docs/library/mdi/guides/brand-icons/) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I have followed the [Material guidelines](https://material.io/design/iconography/system-icons.html) for system icons.

### Tell us about your contribution.

The library already has several icons with a car key, folders and other things, but the infrastructure of the railway transport also does not stand still and many freight transport companies may need an icon for a wagon with a key.

I think icons 1 and 4 best, but I would like to have and the `-outline` icon (2). Option 3 is highly questionable, as it is not displayed very clearly.

### Share a preview of your contribution.

| Numbar | Icon  |
| ------ | ----------------------------------------------------------------------------------------------------------- |
| 1      | ![image](https://github.com/Templarian/MaterialDesign/assets/44722574/a74826ff-1c63-4ae6-a11b-87cf4d28c225) |
| 2      | ![image](https://github.com/Templarian/MaterialDesign/assets/44722574/24d4c990-7e48-40fa-aed6-9b2f666ba822) |
| 3      | ![image](https://github.com/Templarian/MaterialDesign/assets/44722574/2ee9d39a-8edf-4b1d-9caf-fd5350b4809d) |
| 4      | ![image](https://github.com/Templarian/MaterialDesign/assets/44722574/cd494407-13df-48cb-a54b-1a3921c06fc0) |

### Upload the icon source.

[wrench-trains.zip](https://github.com/Templarian/MaterialDesign/files/12369040/wrench-trains.zip)

**Comment by MrGrigri** [comment]
The wrench is a modifier so the `wrench` will go at the end of the name.

**Comment by andrejsharapov** [comment]
Right. I thought that the key in this situation is more important 🤔

**Comment by andrejsharapov** [comment]
I would add this link to the contribution issue_template as check 🤔

**Comment by Templarian** [comment]
Are these really required? They seem a bit niche. Would a more general repair icon make more sense.

**Comment by andrejsharapov** [comment]
Not all. I drew more variants, but you can add one. They are needed because in organizations like mine, maintenance contracts are required to be displayed with a division by type of train/wagon. The wrench icon is too abstract in many ways.

And I would add a wrench icon for the airplane.
